react/src/lib/EventOverlay/index.js

```diff
@@ -472,7 +472,7 @@ class EventOverlay extends React.Component {
     const documentScrollTop = document.documentElement.scrollTop;
     const documentBottom = document.documentElement.scrollHeight;
     const windowBottom = window.pageXOffset + window.innerHeight;
-    const documentRight = document.documentElement.offsetWidth;
+    const documentRight = Math.max(document.documentElement.offsetWidth, document.documentElement.clientWidth);
```

Comment:
The document width is altered by the no-scroll plugin which makes documentRight wrong by a scrollbar width. Using the max of offset and client width stops this happening